lib/theme/app_colors.dart

```diff
@@ -4,4 +4,5 @@ class AppColors {
   static const blackRussian = Color(0xFF15151A);
   static const whiteAlpha18 = Color(0x2EFFFFFF);
   static const whiteAlpha50 = Color(0x80FFFFFF);
+  static const Color nero90 = Color(0xE61E1E1E);
```

Comment:
@Wadeewee Is there a reason to prefer xml files to dart files to keep the colors? 🤔

Reply:
@AVI5HEK The idea like you add the `image` or `font` and let Dart generate `gen.dart` file. by adding the `.xml` You don't need to create an `AppColors` class because Dart will generate a `ColorName` file for you and then to usage you just call `Colorname. nero90`  For example:

```xml
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <color name="blueDiamondColor">#FF532D6D</color>
    <color name="gunPowderColor">#FF505054</color>
    <color name="whitePlatinumColor">#FFE8E8E8</color>
    <color name="silverColor">#FFBEBEBE</color>
    <color name="whiteSmokeColor">#FFF8F8F8</color>
    <color name="magnoliaColor">#FFFAF5FF</color>
</resources>
```
This is the file generate from Dart 👇
```dart
/// GENERATED CODE - DO NOT MODIFY BY HAND
/// *****************************************************
///  FlutterGen
/// *****************************************************

// coverage:ignore-file
// ignore_for_file: type=lint
// ignore_for_file: directives_ordering,unnecessary_import,implicit_dynamic_list_literal,deprecated_member_use

import 'package:flutter/painting.dart';
import 'package:flutter/material.dart';

class ColorName {
  ColorName._();

  /// Color: #FF532D6D
  static const Color blueDiamondColor = Color(0xFF532D6D);

  /// Color: #FF505054
  static const Color gunPowderColor = Color(0xFF505054);

  /// Color: #FFFAF5FF
  static const Color magnoliaColor = Color(0xFFFAF5FF);

  /// Color: #FFBEBEBE
  static const Color silverColor = Color(0xFFBEBEBE);

  /// Color: #FFE8E8E8
  static const Color whitePlatinumColor = Color(0xFFE8E8E8);

  /// Color: #FFF8F8F8
  static const Color whiteSmokeColor = Color(0xFFF8F8F8);
}

```

Reply:
@Wadeewee is there an advantage with this extra step of generating the colors from XML? 🤔

Reply:
It will auto-generate `ColorName` class for you @AVI5HEK